WebApplication/Services/ForgeOSS.cs

```diff
@@ -228,6 +228,23 @@ public async Task<Autodesk.Forge.Client.ApiResponse<dynamic>> GetObjectAsync(str
             return await WithObjectsApiAsync(async api => await api.GetObjectAsyncWithHttpInfo(bucketKey, objectName));
         }
 
+        public async Task<bool> DoesObjectExist(string bucketKey, string objectName)
+        {
+            try
+            {
+                DynamicJsonResponse response = await WithObjectsApiAsync(async api =>
+                {
+                    return await api.GetObjectsAsync(bucketKey, PageSize, objectName, null);
```

Comment:
Well, but we do not have OssBucket here and I do not want to construct it just to check the object existence.

Reply:
I would construct it. I think it would be worth. I would create factory for OssBucket objects. It will help you to resolve all the needed deps from DI.
Your implementation is not correct. And by using OssBucket you will receive code for free.

Reply:
> but we do not have OssBucket here

Right. The `OssBucket` is a wrapper over _this_ class.  I tried to say that `OssBucket` is a preferred way to work with OSS, so I would recommend to use it at "caller" place.

Reply:
ok done.